src/client/pages/OfferNew/utils.ts

```diff
@@ -59,19 +61,17 @@ export const isOfferDataAvailable = (
   return offerData !== null
 }
 
-const getHouseholdSizeFromBundledQuotes = (quotes: BundledQuote[]): number => {
+const getHouseholdSizeFromBundledQuotes = (
+  quotes: BundledQuote[],
+): number | undefined => {
   const quoteDetails = quotes.map((quote) => quote.quoteDetails)
 
   for (const details of quoteDetails) {
     if ('householdSize' in details) return details.householdSize
     if ('coInsured' in details) return details.coInsured + 1
   }
 
-  throw new Error(
-    `quoteDetails ${JSON.stringify(
-      quoteDetails,
-    )} must include one of the following: "householdSize" or "coInsured".`,
-  )
+  return
```

Comment:
Can't throw here since neither `householdSize` nor `coinsured` exists on a car insurance. Sure, we could check before calling `getHouseholdSizeFromBundledQuotes` if any of those two exists, but then there's no need to throw either since one of them will exist every time we call the function.